docs/agents/sematext-agent/kubernetes/installation.md

```diff
@@ -1,7 +1,53 @@
+## Helm Chart
 
+The preferred way to install Sematext Agent is via Helm chart. [Sematext Agent helm chart](https://github.com/helm/charts/tree/master/stable/sematext-agent) is available in official charts repo and it installs the Sematext Agent to all nodes in your cluster via `DaemonSet` resource.
 
-### RBAC
-If your cluster has RBAC enabled add `ClusterRole` and `ClusterRoleBindings`. For `ClusterRoleBindings` you should update `REPLACE_WITH_NAMESPACE` value to match with namespace where the Sematext agent will be installed:
+To install it run the following command:
+
+```sh
+helm install --name sematext-agent \
+  --set containerToken=<YOUR_CONTAINER_TOKEN> \
+  --set infraToken=<YOUR_INFRA_TOKEN> \
+  --set logsToken=<YOUR_LOGS_TOKEN> \
+  --set region=US \
+  stable/sematext-agent
+```
+
+For more details, refer to [helm chart docs](https://github.com/helm/charts/blob/master/stable/sematext-agent/README.md).
+
+## Sematext Operator
+
+The Sematext Operator for Kubernetes provides an easy way to deploy Sematext Agent.
+Sematext Operator is available in [OperatorHub.io](https://operatorhub.io/?keyword=sematext), and in RedHat's OpenShift Container Platform.
+
+To install the operator, run the following command:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/sematext/sematext-operator/master/bundle.yaml
+```
+
+Once installed, you can create `SematextAgent` resource that deploys Sematext Agent to all nodes in your cluster via `DaemonSet` resource:
+
+```yaml
+apiVersion: sematext.com/v1alpha1
+kind: SematextAgent
+metadata:
+  name: test-sematextagent
+spec:
+  region: "US"
```

Comment:
Can we add a comment like: # change to "EU" if you are using Sematext Cloud Europe.